### DISCUSSION TITLE
Form with useFieldArray

### DISCUSSION THREAD

**Discussion by joaopedrodcf** [shadcn-ui/ui#1380] [Q&A] [answered]
I tried to look but I couldn't find an example of the form using the `useFieldArray`, how we would use i ? 

What I found was a reference to this tweet: https://twitter.com/shadcn/status/1659639530734116864

**Comment by joaopedrodcf** [comment, marked as answer]
There is already an example on the website:
<img width="707" alt="image" src="https://github.com/shadcn-ui/ui/assets/11476152/a19d89d6-f791-4979-b27b-39fb3da238b6">

And the code for that can be found here:
https://github.com/shadcn-ui/ui/blob/main/apps/www/app/examples/forms/profile-form.tsx#L72

calling the useForm and the useFieldArray
<img width="283" alt="image" src="https://github.com/shadcn-ui/ui/assets/11476152/1f22efdf-e7d6-4b99-8586-a23e759fef52">

Displaying it and adding elements on click on the button
<img width="461" alt="image" src="https://github.com/shadcn-ui/ui/assets/11476152/074be6e6-2194-4e62-b0e2-cf76384759e9">

Thanks @shadcn  for the reply on twitter/x

**Reply by a8ns** [reply]
Hi @joaopedrodcf, thanks for the Q&A! Do you know how to add multiple FormFields (e.g. in form of a Card) with an append, given a zod schema is defined?

**Reply by joaopedrodcf** [reply]
I don't have an example now but feel free to dm me in Twitter/X if you still need help after 👌

**Reply by MurzNN** [reply]
Sorry for necroposting, but couldn't find anything fresher about the field arrays.

The example link https://github.com/shadcn-ui/ui/blob/main/apps/www/app/examples/forms/profile-form.tsx#L72 is dead and the current Profile form on the page https://www.shadcn.io/patterns/form-patterns-3 misses the array field. 

Could anyone provide a fresh working example with a shadcn form using an array of fields?
